src/libical/icalrecur.c

```diff
@@ -1995,7 +1995,7 @@ static int __day_diff(icalrecur_iterator *impl, icaltimetype a, icaltimetype b)
     return diff;
 }
 
-/** Increment month is different that the other increment_* routines --
+/** increment_month is different than the other increment_* routines --
```

Comment:
should this be @ brief?

Reply:
Yes. I also did that for many other one-liners too.